presto-main/src/main/java/io/prestosql/sql/planner/StatisticsAggregationPlanner.java

```diff
@@ -128,7 +126,7 @@ private ColumnStatisticsAggregation createColumnAggregation(ColumnStatisticType
 
     private ColumnStatisticsAggregation createAggregation(QualifiedName functionName, SymbolReference input, Type inputType, Type outputType)
     {
-        Signature signature = metadata.getFunctionRegistry().resolveFunction(functionName, TypeSignatureProvider.fromTypes(ImmutableList.of(inputType)));
+        Signature signature = metadata.resolveFunction(functionName, TypeSignatureProvider.fromTypes(ImmutableList.of(inputType)));
```

Comment:
The `ImmutableList.of` shouldn't be needed with the new varargs method. Also, static import?